tests/rmw_utils.rs

```diff
@@ -0,0 +1,13 @@
+use rmwrs::utils;
```

Comment:
These 2 tests can go directly into utils.rs, unless you know of any reason not too. I don't have any set guidelines yet about tests, but I think generally if there are only a few tests that it's fine to keep them with the related functions in the related file. When there are a lot of tests, I find it easier to separate them into single tests/file.rs.

Reply:
Added this change in latest commit.